aiohttp/web_runner.py

```diff
@@ -135,6 +136,9 @@ def name(self) -> str:
         self._server = await loop.create_unix_server(
             server, self._path,
             ssl=self._ssl_context, backlog=self._backlog)
+        # create_unix_server sets file permission as rwxr-xr-x, which is not
```

Comment:
@hongquan this assumption is incorrect. `create_unix_server()` doesn't set any privileges. It's the OS kernel which does so following the currently set `umask` value. You can modify the umask in the shell session where you run aiohttp and it file creation syscalls will respect that.

Reply:
Thanks.